docs/quickstart.html

```diff
@@ -215,19 +215,17 @@ <h4 class="anchor-heading">
             Edit the <code class="language-bash">config/connect-standalone.properties</code> file, add or change the <code>plugin.path</code> configuration property match the following, and save the file:
```

Comment:
We don't actually mean it "should match the following" if _the following_ is some bash commands. 

```suggestion
            Edit the <code class="language-bash">config/connect-standalone.properties</code> file, adding or changing the <code>plugin.path</code> configuration property to include <code>libs/connect-file-{{fullDotVersion}}.jar</code> and save the file. Alternatively, in your terminal session you can use the following command:
```